CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -637,6 +754,13 @@ static void __InitTZStrings(void) {
     });
 }
 
+#elif TARGET_OS_ANDROID
+static void __InitTZStrings(void) {
```

Comment:
Why not handle this as Windows and just not use them?

Reply:
I will check the Windows paths again, but I didn’t want to create conditional compilation branches if not needed (Android falls into most of the Linux branches).